client/my-sites/earn/home.tsx

```diff
@@ -471,7 +494,9 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 						components: {
 							a: (
 								<a
-									href="https://wordpress.com/support/video-tutorials-add-payments-features-to-your-site-with-our-guides/"
+									href={ localizeUrl(
+										'https://wordpress.com/support/video-tutorials-add-payments-features-to-your-site-with-our-guides/'
+									) }
```

Comment:
ESLint required addition of the localizeUrl's in this file. Otherwise wouldn't have made that part of the same PR.